### PR TITLE
chore: bump up dashd version for mainnet in dashmate

### PR DESCRIPTION
**Pull request by shotonoff**
## Issue being fixed or feature implemented

Dashmate has wrong dashd version for mainnet configuration

## What was done?

Bump up version from 18.2.2 to 19.0.0

## How Has This Been Tested?

N/A

## Breaking Changes

N/A

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/dashmate/configs/system/mainnet.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const mainnetConfig = lodashMerge({}, baseConfig, {
   },
   core: {
     docker: {
-      image: 'dashpay/dashd:18.2.2',
+      image: 'dashpay/dashd:19.0.0',
```
Comment (strophy) [inline packages/dashmate/configs/system/mainnet.js:19]: Let's go straight to `19.1.0`?

**Review by strophy — CHANGES_REQUESTED** [review]

File: packages/dashmate/configs/system/mainnet.js (hunk 1 above)
Comment (shumkov) [reply packages/dashmate/configs/system/mainnet.js:19]: Agree

Comment (shotonoff) [reply packages/dashmate/configs/system/mainnet.js]: along with the main network, devnet and testnet were updated

**Review by shotonoff — COMMENTED** [review]
update dashd version for mainnet, devnet and testnet

**Review by shumkov — APPROVED** [review]
👍

**Review by pshenmic — APPROVED** [review]
This is good

**Pull request merged**